.travis.yml

```diff
@@ -127,7 +136,13 @@ script:
   # Note: bazel test implies fetch+build, but this gives us timing.
   - bazel fetch //tensorboard/...
   - bazel build //tensorboard/...
-  - bazel test //tensorboard/...
+  - |
+    # Limit tests without TensorFlow
```

Comment:
Small change requested: this shows up in failed builds as

<pre><code><strong>The command "# Limit tests without TensorFlow</strong>
if [ -z "${TF_VERSION_ID}" ]; then
  bazel test //tensorboard/... --test_tag_filters=support_notf
else
  bazel test //tensorboard/...
fi
" exited with 3.</code></pre>

which is a bit less readable than it used to be:

<pre><code><strong>The command "bazel test //tensorboard/..." exited with 3.</strong></code></pre>

Could you please replace this with something like

```yaml
  - |
    # When TensorFlow is not installed, run a restricted subset of tests.
    if [ -z "${TF_VERSION_ID}" ]; then
      test_tag_filters=support_notf
    else
      test_tag_filters=
    fi
  - bazel test //tensorboard/... --test_tag_filters="${test_tag_filters}"
```

so that the actual step that fails just has a `bazel test`?

Reply:
Good idea - changing.